src/pages/gift/accounts/ConnectExtension.js

```diff
@@ -25,21 +25,33 @@ const Connecting = () => {
 };
 
 const DownloadExtension = () => {
-  const extensionLink = 'https://polkadot.js.org/extension/';
+  const polkadotExtensionLink = 'https://polkadot.js.org/extension/';
+  const paritySignerCompanionLink = 'https://parity.link/signer-companion';
```

Comment:
The link seems to redirect to an script or sth:
https://parity-signer-companion.surge.sh/
Can you check it is the correct. I open on chrome